sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/src/configuration.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const SDK_VERSION: string = "1.0.0-beta.2";
+
+/**
+ * @internal
+ *
+ * Keys of known environment variables we look up.
+ */
+export type KnownEnvironmentKey = "AZURE_HTTP_TRACING_DISABLED" | "AZURE_TRACING_DISABLED";
+
+export const environment = new Map<KnownEnvironmentKey, string | undefined>();
+
+if (typeof process !== "undefined" && process.env) {
+  environment.set("AZURE_HTTP_TRACING_DISABLED", process.env.AZURE_HTTP_TRACING_DISABLED);
```

Comment:
In a couple other places we check for both upper and lowercase:

https://github.com/Azure/azure-sdk-for-js/blob/08004eb7ac363a07e2bfae0fc1978c387140401c/sdk/core/core-rest-pipeline/src/policies/proxyPolicy.ts#L41
https://github.com/Azure/azure-sdk-for-js/blob/373589ee6ef6dd8d7fd68a47a87eb46feeb29853/sdk/core/core-http/src/util/utils.ts#L255

According to the NodeJS docs "On Windows operating systems, environment variables are case-insensitive." - but I think that means we can't rely on it for Linux/Mac.

Reply:
I've done the same here for consistency. Thanks!